CHANGELOG.md

```diff
@@ -17,11 +17,16 @@ the detailed section referring to by linking pull requests or issues.
 #### Added
 
 - Added login polling to prevent auto-logout when page is open.
+- Added handling of the additional properties in the case of assets
+- Added viewing of policies option in the Asset Detail Dialog
 
 #### Changed
 
 #### Removed
 
+- Removed asset id from asset details and replaced the same with organization
+  name
+
```

Comment:
- Isn't this an old feature? If so, and the changelog entry was missing, let's move it down to the actual release the change happened in.
- The feature was worded "Asset ID is now less prominent in favor of Organization name", we can keep that.
- It should be in the section "Changed", as we didn't really remove any information. The asset id is still available via the detail dialog.